actionpack/test/dispatch/debug_exceptions_test.rb

```diff
@@ -422,7 +424,7 @@ def call(env)
            "action_dispatch.logger" => Logger.new(output),
            "action_dispatch.backtrace_cleaner" => backtrace_cleaner }
 
-    get "/", headers: env
+    get "/unknown_http_method", headers: env
```

Comment:
If this was already logging an error why does the route name need to change?

Reply:
This whole test is checking [this line](https://github.com/rails/rails/blob/092c547d7fca936c7a1fe39a1bdca6bc8d98940c/actionpack/lib/action_dispatch/middleware/debug_exceptions.rb#L168), in which if the application trace is empty we choose to log the framework trace instead.

In the previous version of the test the exception was an `ActionController::RoutingError` and the test was checking that the framework trace was indeed logged. With this change, the framework trace won't be logged for `ActionController::RoutingError` errors, but we want keep the behaviour for any other exception. So I changed the test to raise an `ActionController::UnknownHttpMethod` error and kept the same assertions.

Reply:
Ok then we need a test for `ActionController::RoutingError` does not output a backtrace.

Reply:
@eileencodes I've added the test to check that the backtrace is not logged in a routing error. Please, let me know what you think.